libs/ui/packages/react/src/components/cta/Button/index.tsx

```diff
@@ -13,7 +13,7 @@ interface BaseProps extends BaseStyledProps, BordersProps {
   ff?: string;
   color?: string;
   backgroundColor?: string;
-  size?: "small" | "medium" | "large";
+  size?: "xs" | "small" | "medium" | "large";
```

Comment:
Do we want to add a 4th button size? Do we have Figma associated, please?

Reply:
Here's are the relevant Figma designs, adding them to the PR description:

https://www.figma.com/design/J9eMb0BxoTcfmtG84KZX17/Stake?node-id=3966-137070&t=9WbiUGt67aAmdcet-1